whitepapers/amount.md

```diff
@@ -81,11 +84,11 @@ The original source for *contract awards* was the Federal Procurement Data Syste
 
 ##Considerations
 
-1.  <b>Use of authoritative data.</b>  The FAR requires Federal agencies to annually verify and validate data reported to the Federal Procurement Data System.  Summary results of this validation and verification are made publicly available via USASpending.gov.  Award amounts for Federal procurement awards will continue to use FPDS-NG as the authoritative source for public display of procurement obligations.  Data for financial assistance is currently submitted by agencies using the FFADS+ file format to the Award Submission Portal (ASP). Current reporting requirements are stipulated in M-15-12 and the ASP User Guide.  Treasury and OMB are working with agencies to develop the architecture for future reporting. Once the Award ID is carried in agency financial systems and there is a linkage between financial data and award data, further policy and business process determinations will need to be made about sourcing obligation amounts from agency financial systems. 
+1.  <b>Use of authoritative data.</b>  The FAR requires Federal agencies to annually verify and validate data reported to the FPDS-NG.  Summary results of this validation and verification are made publicly available via USASpending.gov.  Award amounts for Federal procurement awards will continue to use FPDS-NG as the authoritative source for public display of procurement obligations.  Data for financial assistance is currently submitted by agencies using the FFADS+ file format to the Award Submission Portal (ASP). Current reporting requirements are stipulated in M-15-12 and the ASP User Guide.  Treasury and OMB are working with agencies to develop the architecture for future reporting. Once the Award ID is carried in agency financial systems and there is a linkage between financial data and award data, further policy and business process determinations will need to be made about sourcing obligation amounts from agency financial systems. 
```

Comment:
Same comment re: lower-casing Federal. Looks like we haven't been consistent in previous versions of the white papers! Thus, I don't want to hold up this release, but maybe we can go back and make things consistent as time permits.